### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by xitonix**

### CODE REVIEW

**Review by xitonix — COMMENTED** [review]
asdasd

**Review by xitonix — COMMENTED** [review]
sdfsdf

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ alias scp='sshx -scp'
 ```
 
 Test
+Test2
```
Comment (sandbox-alg) [inline README.md:23]: asda

**Review by sandbox-alg — APPROVED** [review]
asdasd

**Review by sandbox-alg — CHANGES_REQUESTED** [review]
sdfsdf

File: README.md (hunk 1 above)
Comment (xitonix) [reply README.md:23]: asdasd

**Review by xitonix — COMMENTED** [review]
sdfsdf

File: README.md (hunk 1 above)
Comment (sandbox-alg) [reply README.md:23]: sdfsdf

**Review by sandbox-alg — CHANGES_REQUESTED** [review]
sdf

**Review by sandbox-alg — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sandbox-alg) [reply README.md:23]: sdfsdfs

**Review by sandbox-alg — CHANGES_REQUESTED** [review]
sdfsdf

**Review by sandbox-alg — APPROVED** [review]
sdfsdf

**Comment by sandbox-alg** [comment]
Closed

File: README.md (hunk 1 above)
Comment (sandbox-alg) [reply README.md:23]: sdfsdfddf

**Comment by sandbox-alg** [comment]
sdfsdf

**Pull request merged**